pkg/testcase/node.go

```diff
@@ -23,7 +23,7 @@ func TestNodeStatus(
 		expectedNodeCount += cluster.NumWinAgents
 	}
 
-	Eventually(func(g Gomega) {
+	Eventually(func(g Gomega) bool {
```

Comment:
What does the boolean addition do here?

Reply:
adding a way to return me referent logs when the function do not succeed, since i was having timeout all the time